### ISSUE TITLE
[Feature Request] Reward Controlling

### ISSUE THREAD

**Issue by suprememiners** [lorddusk/HQM#31]
One thing I have really seen when using HQM is that when people team rewards become unfair especially any currency styled rewards. 
Using cookies as an example my map had cookies set to a really hard recipe and you get so many to win the game, or unlock other quests.

But setting the option to one reward means that, the cookie rewards are fair but any food that you would give 1 player has to then get split between the team (2+) and when you are relying on that to get your map to work its difficult to get the right balance.

I propose having the reward settings to included an option where the map maker can set wether this can be given out once or to every team member (or to a limited number of them - but thats not as essential)

I hope you like my idea and get around to implementing it. I feel that it would improve the ways in which you can use the mod.

Thanks
~SupremeMiners (Simon)

**Comment by suprememiners** [comment]
Loving the work you guys are doing to the mod at the moment, I am interested to know if this feature is going to get worked on anytime soon?

Thanks

**Comment by way2muchnoise** [comment]
I had a busy few weeks but might revamp the Reward system over the comming week(s). Depending how many issues I run into this might make in. If I can get the code the way it is in my head atm this will make it, but issues can occur.

**Comment by suprememiners** [comment]
Yeah, thats understandable.
Keep me informed, and if my previous explanation wasent that good at explaining what it is I want you to add, just ask.

**Comment by way2muchnoise** [comment]
I started the overhaul of the rewards with #134 , your request will take a bit longer tho as it requires me to rewrite how the claiming works as well. And how teams do things I guess

**Comment by suprememiners** [comment]
Yeah, I reaslised there was a lot that would need to be done. Although I thank you for putting in the time to get this to work. I truly believe it could add a new dimention to map/ quest making.

**Comment by noobanidus** [comment]
The change to specifying which items can be claimed by every team member versus only 1 team member is coming in the rewrite.
